packages/extensions/venia-sample-language-packs/package.json

```diff
@@ -1,16 +1,16 @@
 {
     "name": "@magento/venia-sample-language-packs",
     "version": "0.0.1",
-    "description": "Provides demo translations for PWA Studio.",
     "publishConfig": {
         "access": "public"
     },
-    "license": "(OSL-3.0 OR AFL-3.0)",
+    "description": "Provides demo translations for PWA Studio.",
     "main": "./i18n-intercept.js",
-    "repository": "github:magento/pwa-studio",
     "scripts": {
-      "clean": " "
+        "clean": " "
     },
+    "repository": "github:magento/pwa-studio",
+    "license": "(OSL-3.0 OR AFL-3.0)",
```

Comment:
No changes to this file other than reordering properties. This fixes a `package-json/order-properties` lint failure